sources/packages/web/src/views/aest/institution/Programs.vue

```diff
@@ -175,17 +141,6 @@ export default defineComponent({
         },
       });
     };
-    const pageSortEvent = async (event: any) => {
```

Comment:
As we talked, let's try to have the server-side pagination included in this PR also.